packages/allure-cucumberjs/src/CucumberJSAllureReporter.ts

```diff
@@ -178,6 +177,8 @@ export class CucumberJSAllureFormatter extends Formatter {
     this.currentTest = new AllureTest(this.allureRuntime, Date.now());
     this.currentTest.name = pickle.name;
 
+    this.currentTest?.addLabel(LabelName.HOST, ALLURE_HOST_NAME || os.hostname());
```

Comment:
if dns lookup is performed during `os.hostname()` call we may need to cache hostname into some static variable. Otherwise we will have some significant performance degradation as it's called once per scenario

Reply:
👍